docs/products/kafka/kafka-connect/howto/jdbc-catalog-postgres.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Configure the Iceberg sink connector with a PostgreSQL JDBC catalog
+sidebar_label: PostgreSQL JDBC catalog
+---
+
+import RelatedPages from "@site/src/components/RelatedPages";
+import CreateIcebergSinkConnector from "@site/static/includes/create-iceberg-sink-connector.md";
+
+The JDBC catalog stores Iceberg table metadata in a PostgreSQL database. Use it to
+integrate Apache Kafka, Amazon S3, and a PostgreSQL-based metadata store.
+
+## Prerequisites
+
+- [Aiven for Apache Kafka® service](/docs/products/kafka/kafka-connect/howto/enable-connect)
+  with Apache Kafka Connect enabled, or a
+  [dedicated Aiven for Apache Kafka Connect® service](/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster)
+- Apache Kafka topic that contains the source data
+- Apache Kafka control topic used to coordinate Iceberg table commits
+  (for example, `control-iceberg`)
+- [Aiven for PostgreSQL® service](/docs/products/postgresql/get-started) or another
+  PostgreSQL instance that is publicly accessible
+- Amazon S3 bucket for storing Iceberg table data
+- IAM user or role with permissions to read and write to the S3 bucket
+- AWS access key ID and secret access key for the IAM credentials
+
+
+## Create an Iceberg sink connector configuration
+
+To configure the Iceberg sink connector with the JDBC catalog, define a JSON file that
+uses PostgreSQL for metadata and Amazon S3 for storage.
+
+:::note
+Loading worker properties is not supported. Use `iceberg.kafka.*` properties instead.
+:::
+
+```json
+{
+"name": "<connector-name>",
```

Comment:
This comment refers to all placeholder instances on this PR:
Normally, we use ALL-CAPS for placeholders in the Aiven docs. Please considers using:

```suggestion
"name": "CONNECTOR_NAME",
```

and relevant alternatives for all the other placehlders.